### PR TITLE
BUILDER-124 / BUILDER-149: Connector Content Handler

### PR DESCRIPTION
**Pull request by rgauss**
Added DelegatingContentHandlerImpl and tests

### CODE REVIEW

**Comment by rgauss** [comment]
@CodrinChirica, I'm not sure why I can't add you as a reviewer here.

File: gytheio-commons/src/test/java/org/gytheio/content/handler/DelegatingContentHandlerImplTest.java [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+    private ContentReference contentReferenceFile1a;
+    private ContentReference contentReferenceFile1b;
+    private ContentReference contentReferenceFile1c;
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+    @Before
+    public void setUp() throws Exception
+    {
+        FileProvider fileProviderA = new FileProviderImpl("ontent-handler-a");
```
Comment (tzclucian) [inline gytheio-commons/src/test/java/org/gytheio/content/handler/DelegatingContentHandlerImplTest.java]: Small typo?

**Comment by montgolfiere** [comment]
I've have added Codrin to project -> and then as a reviewer.

**Comment by CodrinChirica** [comment]
Thanks @montgolfiere ! Can you please double check that I also have permissions to commit ? I'm trying to fix the typo in this PR and it get `permission denied`.

**Comment by rgauss** [comment]
> I would also add a suggestion regarding some of the method names that start with get but are not actual getters - replace the prefix so it doesn't confuse IDEs/Spring/future developers ?

Most of those are already defined by the existing interfaces

**Pull request merged**